apple/Sources/FerrostarSwiftUI/Views/Controls/MuteUIButton.swift

```diff
@@ -12,6 +12,7 @@ public struct MuteUIButton: View {
     }
 
     public var body: some View {
+        // TODO: Use NavigationUIButton?
```

Comment:
Noticed that the mute PR didn't use the convention that others did; probably a quick fix.

Reply:
Addressed it #342